src/Controllers/Scheduler/allShows.php

```diff
@@ -11,7 +11,7 @@
 use \MyRadio\ServiceAPI\ServiceAPI;
 use \MyRadio\ServiceAPI\MyRadio_Show;
 
-$shows = MyRadio_Show::getAllShows();
+$shows = MyRadio_Show::getAllShows(1, true);
```

Comment:
looking at this function call from here you wouldn't know what 1 and true mean, so it should have a comment explaining